### ISSUE TITLE
The App is crashing when trying to configure fingerprint unlock

### ISSUE THREAD

**Issue by pcasalinho** [PhilippC/keepass2android#302]
On Android 7.1.1, VerneeX
Settings / Database Settings / Fingerprint Unlock / Enabling
Keepass2Android has stopped

**Comment by realasking** [comment]
Same problem on my redmi4prime markw, with miui9 android 6.0.1...

**Comment by Stimmenhotel** [comment]
On HTC 10, Oreo 8.0.0 using LeeDroid 5.1.0 R139 Rom.
STR:
1. Enabling "Always Allow FP Unlock" within "LeeDroid Tweaks" app.
----(This will enabling FP unlock after reboot, after entering SIM pin, for example, don't know what else)
2. Start K2A, setting FP unlock on "enabled" (both options work)
----(FP unlock works now. **No restart after step 1**)
3. Restart the phone
4. Open K2A, seeing "failed to initialize" message, "please re-enable"
5. Trying to re-enable  fp unlock fpr K2A ---> Crash

Logcat with LegExtreme "per app filter set to K2A"
[Log_2018-03-22_13-12-41.txt](https://github.com/PhilippC/keepass2android/files/1837513/Log_2018-03-22_13-12-41.txt)
Logcat with verbose logging
[Log_2018-03-22_13-25-08.txt](https://github.com/PhilippC/keepass2android/files/1837514/Log_2018-03-22_13-25-08.txt)

In both cases logging was started right before enabling fp unlock and the crash. stopped a few sec. after crash.

**Comment by PhilippC** [comment]
duplicate of https://github.com/PhilippC/keepass2android/issues/155
